integreat_cms/static/src/js/poi_box.ts

```diff
@@ -154,8 +155,9 @@ const removePoi = () => {
     hideSearchResults();
     // Clear the poi form
     hidePoiFormWidget();
-    toggleContactFieldBox(false);
     console.debug("Removed POI data");
+    document.getElementById("poi-address-container").getElementsByTagName("input")[0].value = "-1";
```

Comment:
Am I understanding correctly that you're searching for the poi input field? If so, I think it would be better to get the Element by id, as it already has the id of `poi-query-input`. With the current implementation I see the potential risk, that we at one point in the future we add another input field and then this line would not be correct anymore?

Reply:
The target is another input element in `_poi_address_container.html` but it has an ID too and your suggestion is applied 😸